homeassistant/components/sensor/recorder.py

```diff
@@ -51,6 +58,14 @@
         POWER_WATT: lambda x: x,
         POWER_KILO_WATT: lambda x: x * 1000,
     },
+    DEVICE_CLASS_PRESSURE: {
+        PRESSURE_BAR: lambda x: x / pressure_util.UNIT_CONVERSION[PRESSURE_BAR],
```

Comment:
what is the reason not to use the `convert` function from `pressure_util`? improve performance?

Reply:
Yes, I thought it will lead to redundant error checking.